src/components/SmallDeviceUserMenu.tsx

```diff
@@ -109,6 +115,7 @@ export default function SmallDeviceUserMenu({
   const [open, setOpen] = React.useState(false);
   const anchorRef = React.useRef<HTMLButtonElement>(null);
   const { isProduction } = useEnvironment();
+  const iconLetter = user?.firstName?.charAt(0) || user?.lastName?.charAt(0);
```

Comment:
Should we use the first letter email if user doesn't have name/lastname ?

Reply:
Great question/suggestion @constanzauanini, I think that's probably a good call — will update.

Reply:
Thank you!